src/docs/docusaurus.config.js

```diff
@@ -34,6 +34,18 @@ const config = {
       }),
     ],
   ],
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en', 'fr', 'vi', 'ru', 'uk', 'es', 'zh-cn'],
+    localeConfigs: {
+      en: {
+        htmlLang: 'en-CA',
+      },
+      fr: {
+        htmlLang: 'fr-fr',
```

Comment:
`fr-CA`? 🇨🇦

Reply:
I don't know about the difference between fr-CA and fr-fr. I'm translating this page, so my French is from France

Reply:
I was following this, https://www.fincher.org/Utilities/CountryLanguageList.shtml.